guides-and-standards/standard-public-repos.md

```diff
@@ -0,0 +1,45 @@
+  - For Python projects, dependencies can also be listed in a `requirements.txt` file or in `pyproject.toml` if you are using a package manager like `poetry`.
+  - Caveat: for libraries or packages, it may be more appropriate to specify a minimum version, rather than an exact version, for some dependencies.
+- **No large files**: The repository should not include files larger than 100MB, even if they are necessary for the code to run. If you need to include large files, they should be hosted on Zenodo or an appropriate domain-specific archive (e.g., the SRA) and a link to the files should be included in the README.
+- **No sensitive information**: The repository should not include any sensitive or private information like API keys or access credentials. If the code requires sensitive information to run, it should be stored in a separate configuration file that is not included in the repository. If you accidentally added sensitive information to your repository, please reach out to the software team for help. **It is not sufficient to delete the sensitive information in a subsequent commit**, as the information will still exist in the commit history.
+- **License**: The repository should use the MIT license.
+  - Caveat: make sure that this license is compatible with the licenses of the software/dependencies used in your project (it usually is). Please check with the software team if you are unsure about this.
+
+## Standards for code quality
+
+- Code should be as clear and well-documented as possible. This includes using descriptive variable names and using comments where appropriate to explain complex or non-obvious parts of the code.
+- Code and documentation should be free of absolute paths (e.g. `/home/ubuntu/my_work_dir/` or `~/Desktop/my_work_dir`). Instead, use relative paths (e.g. `../data/`) or provide a way for the user to specify their own local path using a CLI argument, a configuration parameter, or an environment variable.
+- Code should be formatted and linted. The specific tools used for this will depend on the language:
+  - Python repositories should use `ruff` for linting and `ruff format` for formatting. (The [Python template](https://github.com/Arcadia-Science/python-analysis-template) includes this by default.)
+  - R repositories should use `lintr` for linting. (The [R template](https://github.com/Arcadia-Science/r-analysis-template) includes this by default.)
+  - Snakemake repositories should use `ruff` for formatting and linting and `snakefmt` to format snakefiles.
+
+## [Optional] Additional standards for code quality
+
+These additional standards are recommended, as they will make your code more readable, maintainable, and reproducible, but they are not strictly required.
+
+- Variable and function names should be descriptive, accurate, and unambiguous. Avoid unnecessary abbreviations and do not use single-letter names.
```

Comment:
This one seems a little repetitive with the first bullet point in the previous section, so might want to clarify if this is a requirement or optional

Reply:
Yeah, agreed. I clarified that the "optional" requirement is the part about being unambiguous and avoiding abbreviations.